docs/install-and-run.md

```diff
@@ -254,7 +254,7 @@ steps in your setup:
     #import <EarlGrey/EarlGrey.h>
    ```
 
-2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/EarlGrey.swift) and add it to your test bundle. The file contains
+2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/Swift-3.0/EarlGrey.swift) and add it to your test bundle. The file contains
```

Comment:
I think the easy solution is to link `https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/` which will list all the Swift-* folders. Or move to a different folder structure: `files/swift/3.0` and link to `files/swift`